strider/server.py

```diff
@@ -38,9 +38,16 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DESCRIPTION = """
+<img src="/static/favicon.svg" width="300px">
```

Comment:
```suggestion
<img src="/static/favicon.svg" width="200px">
```

This seems a little large - I would rather not have to scroll down to see all of the routes.